src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      // eslint-disable-next-line
+      case 'addProperties': {
+        Object.assign(state, action.extraData);
+      } break;
```

Comment:
There needs to be a reason to disable eslint for certain lines. Here, it is easily fixable, same for the rest of cases.
```suggestion
      case 'addProperties':
        Object.assign(state, action.extraData);
        break;
```